pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
fwiw, I can merge it myself, after it has been given :+1: by someone else.
See also #62.